scripts/generate_report.sh

```
@@ -80,152 +115,177 @@ STATS=$(gh api graphql -f query='
-log "Fetching PR statistics..."
-# 全リポジトリのPR情報を取得
-PR_QUERY=$(gh api graphql -f query='
-  query($repoOwner: String!, $repoName: String!) {
-    repository(owner: $repoOwner, name: $repoName) {
-      pullRequests(first: 100, states: [OPEN, CLOSED, MERGED], orderBy: {field: CREATED_AT, direction: DESC}) {
-        nodes {
-          createdAt
-          mergedAt
-          state
+# オーガニゼーション別にPRを取得
+for org in $ORGS
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)